templates/users/new.rs.html

```diff
@@ -15,9 +15,10 @@ <h1>@i18n!(ctx.1, "Create an account")</h1>
             @input!(ctx.1, password (password), "Password", form, errors.clone(), "minlenght=\"8\"")
             @input!(ctx.1, password_confirmation (password), "Password confirmation", form, errors, "minlenght=\"8\"")
 
-            <input type="submit" value="@i18n!(ctx.1, "Create account")" />
+            <input type="submit" value="@i18n!(ctx.1, "Create your account")" />
         </form>
     } else {
-        <p class="center">@i18n!(ctx.1, "Sorry, but registrations are closed on this instance. Try to find another one")</p>
+<p class="center">@i18n!(ctx.1, "Apologies, but registrations are closed on this instance.
```

Comment:
The closing parenthesis for this i18n call should be at the end of this line, not before the closing `</p>`, nested call won't compile